Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM golang:1.10.1-alpine3.7 as go-builder
```

Comment:
Why are we using the `as` operator here? I think change of the base image should be an intrusive change.

Reply:
The `as` is just to name stages. In this case, the first stage is called `go-builder`. We can use that reference to obtain files from a different stage.

Reply:
Thanks for the explanation.